bcda/api_test.go

```diff
@@ -31,14 +39,225 @@ func (s *APITestSuite) SetupTest() {
 	s.rr = httptest.NewRecorder()
 }
 
+func (s *APITestSuite) TestBulkRequest() {
+	s.SetupAuthBackend()
+
+	acoID := "0c527d2e-2e8a-4808-b11d-0fa06baf8254"
+	user, err := s.AuthBackend.CreateUser("api.go Test User", "testbulkrequest@example.com", uuid.Parse(acoID))
+	if err != nil {
+		s.T().Error(err)
+	}
+
+	token := jwt.New(jwt.SigningMethodRS512)
+	token.Claims = jwt.MapClaims{
+		"sub": user.UUID.String(),
+		"aco": acoID,
+		"id":  uuid.NewRandom().String(),
+	}
+	token.Valid = true
```

Comment:
This is a weird usage.  I think this is normally set internally when the token is parsed/verified

Reply:
I thought so too, but it didn't seem to get set when creating it manually